src/testing/testhelpers/integ.py

```diff
@@ -83,6 +83,8 @@ def assertions(r, endpoint, schemas_path):
     if r.headers["Content-Type"].startswith("application/json") or r.headers["Content-Type"].startswith("text/plain"):
         assert validate_schema(r.text, endpoint, schemas_path) is True
     else:
+        print('### Header does NOT contain expected Content-Type, response object = ')
+        print(r)
```

Comment:
Many tests in `orders` and `users` fail on the next line (`assert False`)
I add some info for ease of debugging.